sdks/python/container/Dockerfile

```diff
@@ -37,11 +37,11 @@ RUN apt-get update && \
 ####
 # Install required packages for Beam Python SDK and common dependencies used by users.
 ####
-
+ARG install_release_candidates
```

Comment:
this ARG has no knowledge of what you pass; you can call it `pip_extra_options`  for example.

Reply:
does it need a default value?

Reply:
yes. but i defined the default value to be null in common.gradle file. I can provide default here